libcodechecker/analyze/analyzers/analyzer_types.py

```diff
@@ -22,6 +21,7 @@
 from libcodechecker.analyze.analyzers import result_handler_clang_tidy
 from libcodechecker.analyze.analyzers import result_handler_plist_to_db
 from libcodechecker.analyze.analyzers import result_handler_plist_to_stdout
+from libcodechecker.cmd.cmd_line_client import setup_client
```

Comment:
Or maybe such code should be pulled out from the cmd_line_client into a separate helper module to avoid layering violations.